bin/npg_publish_tree.pl

```diff
@@ -63,7 +67,8 @@
            'metadata=s'                          => \$metadata_file,
            'restart-file|restart_file=s'         => \$restart_file,
            'source-directory|source_directory=s' => \$source_directory,
-           'verbose'                             => \$verbose);
+           'verbose'                             => \$verbose,
+           'mlwh_json=s'                         => \$mlwh_json_filename);
```

Comment:
This supports a CLI option with underscores (`--mlwh_json`), but not dashes e.g. (`--mlwh-json`). For consistency with other options in this and other scripts, it should support both. It saves the user hitting `SHIFT` when typing. The options should be maintained in alphabetical order to help people find them.

Reply:
Ok I will add both and move them after the 'metadata' option